omod/src/main/java/org/openmrs/module/webservices/rest/Activator.java

```diff
@@ -28,6 +29,9 @@
 	@Override
 	public void started() {
 		log.info("Started the REST Web Service module");
+		
+		// Cache Swagger Spec
+		new SwaggerSpecificationCreator().getJSON();
```

Comment:
Doesn't the above lead to an even slower start of openmrs? Why not do it on the first call? That way, those who will never ask for swagger docs do not get affected.

Reply:
Yeah I understand. It'll add up about 2 seconds to the startup time. However if we cache it there, load time for the docs page will be instantaneous. Please make the final call :) I'll edit then.

Reply:
Putting performance aside, i do not see any reason why we do not just do it the first time one tries to access the docs.

Reply:
Got it.